src/test/resources/features/completion/wildcards_completion.feature

```diff
@@ -19,8 +19,12 @@ Feature: Completion for wildcards
       cwl: "{non-wildcard2}"
       script: "{non-wildcard3}"
       shell: "{non-wildcard4}"
+
+
+    use rule NAME as NAME2 with:
+      input: "use{}"
     """
-    When I put the caret after input: "{
+    When I put the caret after input: "<addition>{
```

Comment:
Let's make test easier to read, replace:
`after input: "<addition>{` 
with:
`after <signature>`

P.S: multiple times in this file